.github/workflows/deploy.yml

```diff
@@ -5,20 +5,33 @@ on:
     branches:
       - main
 
+    workflow_dispatch:
```

Comment:
Manually workflow can be dispatched. 
<img width="671" alt="image" src="https://user-images.githubusercontent.com/45073703/220052554-4442bc26-5832-4c81-a6db-82b6b239b90b.png">